main.py

```diff
@@ -372,7 +372,10 @@ def user_token(self, userid='core'):
     def check_docker(self):
         '''Check docker services status'''
         stack = 'docker-cloud stack inspect %s' % self.stackname
-        self._run(stack)
```

Comment:
@Mikanebu it not only prints but returns as well. so you could 
```
json.loads(out)
# and do the stuff
```
But if you think this is better way let's keep it